standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -5836,11 +5835,11 @@ private void removeTxnsFromMinHistoryLevel(Connection dbConn, List<Long> txnids)
     }
   }
 
-  private synchronized static DataSource setupJdbcConnectionPool(Configuration conf, int maxPoolSize, long getConnectionTimeoutMs) {
+  protected synchronized static DataSource setupJdbcConnectionPool(Configuration conf, int maxPoolSize) {
```

Comment:
Could we add @Override if we override parent method?

Reply:
I'm not overriding this method, just changed the visibility and removed getConnectionTimeoutMs param as it was not used.